go/lesson03/README.md

```diff
@@ -35,7 +35,7 @@ Execute an HTTP request against the formatter:
 
 ```
 $ curl 'http://localhost:8081/format?helloTo=Bryan'
-Hello, Bryan!%
```

Comment:
This is how curl prints the output on Mac. What the `$` for?

Reply:
When I tested it on Linux, this is what I saw:
```
$ curl 'http://localhost:8081/format?helloTo=Bryan'
Hello, Bryan!$
```
The response has no newline, so the next character is the shell prompt which in your example is the `$` so I figured the example should be reflective of that.

Perhaps your prompt is a `%`?